pages/[network]/curators/index.tsx

```diff
@@ -1,18 +1,64 @@
 import React from "react";
 
-import {serverSideTranslations} from "next-i18next/serverSideTranslations";
-import {GetServerSideProps} from "next/types";
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useRouter } from "next/router";
+import { GetServerSideProps } from "next/types";
 
-import ReadyToPropose from "./ready-to-propose";
+import CouncilLayout from "components/council-layout";
+import CuratorsList from "components/curators-list";
+import ListIssues from "components/list-issues";
 
 export default function PageCouncil() {
-  return <ReadyToPropose/>
+  const { t } = useTranslation(["council"]);
+  const router = useRouter();
+  const { type } = router.query;
+
+  const types = {
+    "curators-list": <CuratorsList key={"curators-list"} />,
+    "ready-to-close": (
+      <ListIssues
```

Comment:
add a `inView: boolean` prop to this component that _will not_ render or start useEffects if it's not true -- as long as it exists:

```
// on this object, add
inView={type === "ready-to-close"}


// on ListIssues component, add the following pseudo-logic
{inView: null}
// ...
if (inView !== null && inView === false)
  return;